app/models/repository_attributes.rb

```diff
@@ -1,6 +1,6 @@
 class RepositoryAttributes < ActiveRecord::Base
   belongs_to :user
-  validates :repository_id, presence: true
+  validates :repository_id, presence: true, uniqueness: { scope: :user_id}
```

Comment:
Thinking about this once more I don't believe we want to allow multiple project owners for now. Allowing that kind of behaviour may lead to expectations about features we do not yet implement (such as adding members to the project).

So maybe it's better to leave the scope out.